common-files/constants/bigInts.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-extend-native */
+BigInt.prototype.toJSON = function () {
```

Comment:
Looks like we don't use this `toJSON` at all in the codebase. Good time to get rid of it.

Reply:
I already spoke about it with @IlyasRidhuan, this comment is not relevant anymore. Sorry